src/app/city-search/city-search.module.ts

```diff
@@ -1,17 +1,11 @@
 import { CommonModule } from '@angular/common';
 import { NgModule } from '@angular/core';
 import { FormsModule } from '@angular/forms';
-import { BrowserModule } from '@angular/platform-browser';
```

Comment:
Imported in `CommonModule` already, so no need to re import.
PS i think `FormsModule` can also be removed as its also in the CommonModule